### ISSUE TITLE
getKeys() throws ConsulException

### ISSUE THREAD

**Issue by bprager** [OrbitzWorldwide/consul-client#130]
Gentlemen,
I am having trouble in using getKeys().
The following code throws a ConsulException:
```
public class Test {

	private static final Logger logger = Logger.getLogger(Test.class);

	public static void main(String[] args) {
		String hostname = "192.168.99.100";
		Integer port = 8500;
		String key = "my_key";
		// get consul client
		HostAndPort hostAndPort = HostAndPort.fromParts(hostname, port);
		Consul consul = Consul.builder().withHostAndPort(hostAndPort).build();
		KeyValueClient kvClient = consul.keyValueClient();
		// simple string test
		String testString = "Hello World!";
		logger.info("write: '" + testString + "'.");
		kvClient.putValue(key, testString);
		// check keys (this line throws com.orbitz.consul.ConsulException: Consul request failed)
		List<String> list = kvClient.getKeys(key);
		logger.info(list.size() + " keys found.");
	}
}

```

**Comment by rickfast** [comment]
I'll take care of this sometime this week. Thanks

**Comment by rickfast** [comment]
fixed by 5f4214c7f11b14a7c113f3d6b5886b28bf5d5b02
